romancal/source_detection/source_detection_step.py

```diff
@@ -193,7 +192,9 @@ def process(self, input):
 
             # tweakreg_catalog_name will be saved to the final output file,
             # while tweakreg_catalog is intended to be deleted by TweakRegStep
-            input_model.meta["source_detection"] = maker_utils.mk_source_detection()
+            input_model.meta["source_detection"] = rdm.WfiImageModel.make_default(
+                _shrink=True
+            ).meta.source_detection
```

Comment:
Correct `_shrink=True` limits the image arrays created to `(8, ...)`. Since this is only there to set `source_detection` (which I think might be redundant) and `source_detection` contains no arrays, this is just to make things cheaper.